### ISSUE TITLE
[Help] 关于readme中优化显存占用的描述

### ISSUE THREAD

**Issue by xyfZzz** [THUDM/ChatGLM2-6B#30]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

请问下这句话怎么理解：
“此外，ChatGLM2-6B 采用 Causal Mask 进行对话训练，连续对话时可复用前面轮次的 KV Cache，进一步优化了显存占用。”，

为什么说采用 Causal Mask 进行对话训练，连续对话时可复用前面轮次的 KV Cache能够优化显存占用？
谢谢！

### Steps To Reproduce

请问下这句话怎么理解：
“此外，ChatGLM2-6B 采用 Causal Mask 进行对话训练，连续对话时可复用前面轮次的 KV Cache，进一步优化了显存占用。”，

为什么说采用 Causal Mask 进行对话训练，连续对话时可复用前面轮次的 KV Cache能够优化显存占用？
谢谢！

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by duzx16** [comment]
因为使用 causal attention 的时候后面的token不会影响到前面token的计算，所以可以直接重用前面的 kv_cache，而不用重新计算整个对话历史
